src/dropdown.test.interactions.filterable.ts

```diff
@@ -1429,8 +1429,10 @@ it('shows an ellipsis when the label of its selected option is set programmatica
   option.label = 'x'.repeat(500);
   await host.updateComplete;
 
-  const ellipsis = host.shadowRoot?.querySelector('[data-test="ellipsis"]');
+  // Wait for the resize observer to do its thing.
+  await aTimeout(0);
```

Comment:
@ynotdraw: A fix for that flaky test you mentioned a while back.

Reply:
Thanks!